fastlane/fastlane/Fastfile

```diff
@@ -178,7 +186,7 @@ private_lane :print_release_changelog do |options|
   tool_name = options[:tool]
   old_version = options[:old_version]
 
-  cmd = "git log --pretty='* %B' #{tool_name}/#{old_version}...HEAD --no-merges .."
+  cmd = "git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges .."
```

Comment:
Here is github is fine
On Apr 15, 2016 2:18 PM, "Michael Furtak" <notifications@github.com> wrote:

> In fastlane/fastlane/Fastfile
> <https://github.com/fastlane/fastlane/pull/4230#discussion_r59917557>:
>
> > @@ -178,7 +186,7 @@ private_lane :print_release_changelog do |options|
> >    tool_name = options[:tool]
> >    old_version = options[:old_version]
> >
> > -  cmd = "git log --pretty='* %B' #{tool_name}/#{old_version}...HEAD --no-merges .."
> > +  cmd = "git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges .."
>
> Comment here, or in the code?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/fastlane/fastlane/pull/4230/files/b76ea60104b97338ae32042c8e068c014a3e71d2#r59917557>
>